frontend/web/components/segments/Rule/components/RuleConditionRow.tsx

```diff
@@ -75,7 +75,7 @@ const RuleConditionRow: React.FC<RuleConditionRowProps> = ({
 
   const ALLOWED_CONTEXT_VALUES: OptionType[] = [
     {
-      enabled: operator === 'PERCENTAGE_SPLIT',
+      disabled: operator !== 'PERCENTAGE_SPLIT',
```

Comment:
Ok, got it - so this Identity key is only usable by percentage split. 2 things

1 - If I switch operator with this selected it doesn't clear and shows $.identity.key in the select
2 - I think I mentioned this before but would it make more sense to label this "Identifier + Environment Name" People don't know what identity key is. I'd actually say the same for "identifier" too - we don't show this term anywhere. Maybe it should be the heading in the identity table / show "Identifier: ''" in the detail page title.